history/peter-toma.md

```diff
@@ -0,0 +1,19 @@
+---
+description: Developer of the first machine translation product
+---
+# Peter Toma
+Peter Toma was a computer scientist and linguistics researcher. 
+
+In 1956, Toma developed software for machine translation at the California Institute of Technology. 
+The system translated Russian texts into English. Later, he worked at Georgetown University where the system 
+Serna was developed for the same translation direction. 
+
+In 1962, he founded a company which created the systems Autotran and Technotran. 
+They were specialized in Russian-to-English translation in the fields of atomic energy and medicine.  
+
+In the mid-1960s, with the support of the German Research Foundation Toma programmed the [Systran](/companies/systran.md) prototype. 
```

Comment:
> In the mid-1960s, Toma programmed the [Systran](/business/companies.md#systran) prototype, with the support of the German Research Foundation.

Or at least put a comma after *Foundation*...  Otherwise it becomes hard to parse.

Also the link should be to `/business/companies.md#systran` - because there is not such page